<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Jeuxcarte</class>
 <widget class="QMainWindow" name="Jeuxcarte">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1762</width>
    <height>860</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Jeuxcarte</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-repeat: no-repeat;
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-repeat: no-repeat;
</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>260</y>
      <width>201</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>691</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout"/>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>220</y>
      <width>131</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>281</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="labelScoreJoueur1">
       <property name="styleSheet">
        <string notr="true">font: 900 14pt &quot;Segoe UI Black&quot;;

color: rgb(255, 255, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelScoreOrdinateur">
       <property name="styleSheet">
        <string notr="true">font: 900 14pt &quot;Segoe UI Black&quot;;

color: rgb(255, 255, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>210</y>
      <width>161</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3"/>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>210</y>
      <width>160</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4"/>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>450</y>
      <width>701</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2"/>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>-20</y>
      <width>1511</width>
      <height>1051</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-repeat: no-repeat;
</string>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="EasyButton">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>330</y>
        <width>371</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);

font: 16pt &quot;Segoe Print&quot;;
background-color: transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>420</y>
        <width>371</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background-color: transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QStackedWidget" name="stackedWidget_2">
      <property name="geometry">
       <rect>
        <x>-30</x>
        <y>10</y>
        <width>1721</width>
        <height>861</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);

font: 16pt &quot;Segoe Print&quot;;
background-color: transparent;</string>
      </property>
      <widget class="QWidget" name="page_3">
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>330</y>
          <width>501</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);

font: 16pt &quot;Segoe Print&quot;;
background-color: transparent;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_4"/>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1762</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
